packages/cli/src/utils/pull.js

```diff
@@ -0,0 +1,49 @@
+// prob put this in files
+const fs = require('fs');
+const path = require('path');
+
+const constants = require('../constants');
+const { respectGitIgnore } = require('./build');
+
+const isBlacklisted = (filePath) => {
+  return constants.BLACKLISTED_PATHS.find((excluded) => {
```

Comment:
While we're at it, let's rename the constant to something like `EXCLUDED_PATHS`.

Reply:
Ah nice I had the same thought.